ckan/templates/user/snippets/pager.html

```diff
@@ -0,0 +1,16 @@
+{% extends "snippets/pager.html" %}
+
+{% block pager %}
+    {% if page.page_count > 1 %}
+        <div class='pagination pagination-centered'>
+            {{ page.pager(
+                q=q,
+                order_by=order_by,
+                symbol_previous=symbol_previous,
+                symbol_next=symbol_next,
+                pager_item_text=pager_item_text
+                )
+            }}
+        </div>
+    {% endif %}
+{% endblock %}
```

Comment:
isn't this block the same as the one in the extended template? If so, why redefine it? (same question for the other pager snippets)